### PR TITLE
update TM info; seL4 TM is registered in US

### PR DESCRIPTION
**Pull request by bbrcknl**
Signed-off-by: Birgit Brecknell <birgit@sel4.systems>

### CODE REVIEW

File: content/Foundation/Trademark/home.pml [hunk 1]
```diff
@@ -12,8 +12,10 @@
   in your work, following the usage rules below.
 </p>
 <p>
-  The Linux Foundation Projects (LF Projects, LLC) has applied to register the
-  seL4 trademark in the United States and other countries.
+   The Linux Foundation has registered seL4<sup>&reg;</sup> as a
+  trademark in the United States. Please use &reg;
```
Comment (lsf37) [inline content/Foundation/Trademark/home.pml:16]: I would say "in the United States and other countries"

**Comment by bbrcknl** [comment]
@gernotheiser The main wording change I found in the trademark home.pml was "Please use &reg;
  with the first mention of seL4 in all materials," in the second paragraph.

File: content/Foundation/Trademark/home.pml [hunk 2]
```diff
@@ -12,8 +12,10 @@
...
+  with the first mention of seL4 in all materials.
+  The Linux Foundation Projects (LF Projects, LLC) has also applied to register the seL4 trademark in other countries.
```
Comment (lsf37) [inline content/Foundation/Trademark/home.pml:18]: Probably don't need that sentence any more if we add "other countries" to the previous one.

Comment (bbrcknl) [reply content/Foundation/Trademark/home.pml:18]: Ok. Gernot said there was some other required wording in the history of the pml, but I didn't see anything else that we might need to add.

Comment (lsf37) [reply content/Foundation/Trademark/home.pml:18]: Ok, let me check

Comment (lsf37) [reply content/Foundation/Trademark/home.pml]: I don't see anything else in the history either.

File: content/Foundation/Trademark/home.pml [hunk 3]
```diff
@@ -12,8 +12,8 @@
   in your work, following the usage rules below.
 </p>
 <p>
-  The Linux Foundation Projects (LF Projects, LLC) has applied to register the
-  seL4 trademark in the United States and other countries.
+   The Linux Foundation has registered seL4<sup>&reg;</sup> as a
+  trademark in the United States. The Linux Foundation Projects (LF Projects, LLC) has also applied to register the seL4 trademark in other countries.
```
Comment (lsf37) [inline content/Foundation/Trademark/home.pml:16]: Ah, sorry, misunderstanding. I meant we can remove: "The Linux Foundation Projects (LF Projects, LLC) has also applied to .." .

The first sentence should instead be: "The Linux Foundation has registered seL4<sup>&reg;</sup> as a   trademark in the United States and other countries".

I'm easy on whether to include the "Please use TM sign" sentence or not. It is in the guidelines that are being linked in the footer, but if people care a lot about it, I'm fine to repeat it in the footer itself.

Comment (bbrcknl) [reply content/Foundation/Trademark/home.pml:16]: Ok. First sentence updated.
I'll leave out the sentence about using the TM sign. It was commented out in the pml, so I'll leave it as a comment unless someone objects.

**Review by lsf37 — APPROVED** [review]
:+1:

**Pull request merged**